src/store/index.js

```diff
@@ -691,20 +740,21 @@ export function sendCssToFrame(value, currentField) {
           ? currentField.name === 'Accordion'
             ? `:not([data-collapse-id="${localSavedWidgetFound ? localSavedWidgetFound.id : savedWidgetFound.id}"]) `
             : `:not([data-id="${localSavedWidgetFound ? localSavedWidgetFound.id : savedWidgetFound.id}"]) `
-          : ''
+          : '';
+
+      const preparedStyles = prepareStyles(style, value, widgetSelector, currentField);
 
-      const preparedStyles = prepareStyles(style, value, widgetSelector, currentField)
       preparedStyles.forEach((styles) => {
-        cssProperties.push(styles)
+        cssProperties.push(styles);
       })
     })
   })
 
   if (currentField.name === 'containerBackgroundColor') {
     _.forEach(cssProperties, (css, index) => {
       if (_.has(css.properties, 'background-color')) {
-        cssProperties[index].properties['background-image'] = 'none'
-        return false
+        cssProperties[index].properties['background-image'] = 'none';
+        return false;
```

Comment:
ESLint should be adding an empty line before this `return`. Why is it not here?

Reply:
What was the reason that this wasn't added before?

Reply:
@tonytlwu This arose after the update of the branch. This branch was merged with the master which contained the code from the previous task